.rubocop.yml

```diff
@@ -28,6 +28,16 @@ Metrics/LineLength:
   Max: 100
   Exclude:
     - Gemfile
-
+  
 Style/Documentation:
   Enabled: false
+
+
+Style/FormatString:
+  Enabled: false
+Style/DotPosition:
+  Enabled: false
+Style/TrailingWhitespace:
+  Enabled: false
```

Comment:
Should we really be disabling these things? The first two are trivial to write in a 'compliant' way and removes the discussion about where dots should go, and the last one means that you won't get a load of "not really changed" lines the next time somebody with a "trim trailing whitespace on save" setting in their editor (which most of us use) saves a file.